resonate/bridge.py

```diff
@@ -204,8 +197,16 @@ def _stop_no_join(self) -> None:
 
     @exit_on_exception
     def _process_cq(self) -> None:
+        shutdown_error: ResonateShutdownError | None = None
         while item := self._cq.get():
             cmd, future = item if isinstance(item, tuple) else (item, None)
+            match cmd, future:
```

Comment:
I think if `shutdown_error is not None` and `future is not None` we should set_exception regardless of the command.